mlos_bench/mlos_bench/launcher.py

```diff
@@ -257,8 +257,8 @@ def _expand_vars(self, value: Any) -> Any:
             # new set of params explicitly.
             # Since this operates by updating the global_config along the way,
             # we need to continually update the set of params to use for substitution.
-            params = dict(os.environ)
-            params.update(self.global_config)
+            params = self.global_config.copy()
+            params.update(dict(os.environ))
```

Comment:
just
```suggestion
            params.update(os.environ)
```
if mypy doesn't mind

Reply:
Answered in #533